local modId  = geode.Mod.getID()
local modVer = geode.Mod.getVersion()

imgui.onDraw(function()
    imgui.window("Video Importer", function()
        imgui.text(modId .. " v" .. modVer)
        imgui.separator()
        imgui.text("Select one object in the")
        imgui.text("editor, then click the")
        imgui.text("import button in Edit tab.")
        imgui.separator()
        imgui.text("Keep GIFs small.")
        imgui.text("emotesizer.com/tools/resize")
    end)
end)

print("Video Importer loaded " .. modVer)